dvc/lock.py

```diff
@@ -15,6 +15,12 @@
 
 DEFAULT_TIMEOUT = 5
 
+FAILED_TO_LOCK_MESSAGE = (
+    "cannot perform the command because another `dvc` process seems to be "
```

Comment:
question to @jorgeorpinel  - is it a regular practice to use backticks in command line? Especially not sure about `dvc` . It seems to me that we should not use markup if the client (command line) does not know how to render it. Probably it's a good idea to highlight path somehow. I would take look at how other popular command line tools do this.

Reply:
Oh yeah we have lots of back ticks in the output strings. @efiop and I talked about it in PR #2644 and decided to use them everywhere for file names, commands, command options, etc. So I think \`dvc\` qualifies.

> Probably it's a good idea to highlight path somehow

Yeah this could be a good substitution although not all terminals support colors but maybe you want to add it to #2784? Or should I add a separate issue to discuss this?

Reply:
kk, it qualifies if it's an exact process name. Though, process name sometimes is not `dvc` it can be python. So I would just use DVC process name in a sense that it's a process created by one of the DVC commands.

I'm still not convinced that it's a good practices to use back-ticks in CLI UI. CLI UI should use plain english with common regular rules. For example it should not include jso or XML or html or Markdown for that sake. I would still check how do other tools deal with this.

Reply:
I see. OK well I guess "DVC process" is more general and correct then. Sorry @algomaster99 could you update that please?

Reply:
> I'm still not convinced that it's a good practices to use back-ticks in CLI UI...

You may be right, I don't know if these characters can cause problems in some terminals. I'm extracting this to #2793.

> it should not include jso or XML or html or Markdown...

In this case we just use back ticks because we're used to them, but not because we're trying to output Markdown. But anyway, yeah let's discuss in the other issue? 🙂

Reply:
Agreed! Missed that discussion somehow. And actually not sure that have a very strong opinion either. I would just if can cause some problems (doubt it) and if there are other good examples on how people handle this (if they do any special at all?).